triagebot.toml

```diff
@@ -493,6 +493,9 @@ libs = [
 ]
 bootstrap = [
     "@Mark-Simulacrum",
+    "@jyn514",
```

Comment:
Please do not add me to this list, I don't have capacity for reviews for the next ~6 months. I am willing to mentor you on reviewing PRs you're assigned to if you find that helpful.